build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20-buster
+FROM golang:1.20.7-bookworm
```

Comment:
there is one also in the integrations tests, but I am not sure that you left it out on pourpose

Reply:
I simply forgot it, let's keep them consistent: [7758c7d](https://github.com/newrelic/nri-haproxy/pull/112/commits/7758c7d55c1d76b03dd0a28ad4a7935f159907ea)